_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -296,6 +296,62 @@ Use the following steps to register a snapshot repository backed by an Azure sto
    }
    ```
 
+### Set up Microsoft Azure Blob Storage
```

Comment:
I think this needs to be reconciled with the "### Registering an Azure storage account" section that immediately proceeds this one. The previous section starts with "Use the following steps to register a snapshot repository backed by an Azure storage account for an OpenSearch cluster deployed using Helm." and this one starts with "To use Microsoft Azure Blob Storage as a snapshot repository, install the `repository-azure` plugin on all nodes...".

I don't actually have a super specific suggestion for how to re-work this, but I think it is potentially confusing and/or partially redundant as written. Parts of the difference are related to using Docker (above) whereas this one gives more generic commands not specific to Docker.